src/World3/world3/plots.jl

```diff
@@ -646,7 +1004,71 @@ fig_27(; kwargs...) = @info "This figure is not implemented yet."
     output per capita IOPC to decline. The added costs incurred by the continued implementation of new technologies even after IOPC has peaked
     force IOPC to fall more precipitously than in Run 7-21.
 """
-fig_30(; kwargs...) = @info "This figure is not implemented yet."
+function fig_30(; kwargs...)
```

Comment:
Besides some bumps related to #79, the generated plot corresponds with the book :heavy_check_mark: